docs/content/items.md

```diff
@@ -215,43 +222,43 @@ To assign an asset as a thumbnail, click on the Select button in the main work a
+Once you have created an item, you can edit it at any time, either by clicking the edit icon (pencil button) or clicking on the item's title and then clicking the "Edit" button in the upper right-hand corner of the screen.
 
-To cancel out of editing, click the Cancel button in the upper right corner, between the Delete and Save buttons. 
+To cancel out of editing, click the "Cancel" button in the upper right corner, between the "Delete" and "Save" buttons. 
 
-![Detail view of the top row if information and buttons on an item edit view. A red arrow points to the Cancel button, on the far right](contentfiles/items_cancel.JPG)
+![Detail view of the top row if information and buttons on an item edit view. A red arrow points to the "Cancel" button, on the far right](contentfiles/items_cancel.JPG)
 
 Editing options are the same as when creating a new item, with the addition of the ability to add any media which has been saved to an item as a property:
 
 ### Media as property 
-Once you have created an item and added media to it, you will also have the option to use media *attached to that item* for a property. 
+Once you have created an item and added media to it, you will also have the option to use media **attached to that item** for a property. 
 
 ![A blue outline of a rectangle highlights resource options for Items, Item sets, and Media to be used as the input for the element "Description"](contentfiles/items_addmediaresource.png)
 
-To use a media resource for a property, select the media option for the element (1). The drawer which opens will display all media attached to the item. Select the media which you want to use(2); this will switch the drawer to just that media (second image). Click the *Select Resource* button at the bottom of the drawer to complete the process (3).
+To use a media resource for a property, select the media option for the element (1). The drawer which opens will display all media attached to the item. Select the media which you want to use (2); this will switch the drawer to just that media (second image). Click the "Select Resource" button at the bottom of the drawer to complete the process (3).
 
 ![Focus on the editing portion of a window for the item "Sense and Sensibility", with the elements Title and Description visible. On the right side of the window, a vertical rectangle (the drawer) is open on the right displaying five media which are attached to the item, including video, images, and text. This image contains steps 1 and 2.](contentfiles/items_mediaresource1.png)
 
 ![The same window as before, but now the drawer on the right displays a thumbnail of the media "Sense and Sensibility DVD Trailer" with a dark gray button labeled "select resource" at the bottom of the drawer area. This image contains step 3](contentfiles/items_mediaresource2.png)
 
 
-## Batch actions
+## Batch editing
```

Comment:
We tend to think about batch deleting as being separate from batch editing, maybe justifying the prior more generic heading here